libs/wingsdk/src/cloud/api.ts

```diff
@@ -441,6 +489,11 @@ export abstract class Api extends Resource {
         `Endpoint for path '${path}' and method '${method}' already exists`
       );
     }
+    if (this._isPathAndMethodAmbiguous(path, method)) {
+      throw new Error(
+        `Endpoint for path '${path}' and method '${method}' is ambiguous`
+      );
```

Comment:
I think it may be more helpful to include the two ambiguous paths in the error message